release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -107,6 +107,13 @@ With this release, you can deploy a cluster on {gcp-short} using the link:https:
 
 With this release, the installation program uses Cluster API instead of Terraform to provision cluster infrastructure during installations on {gcp-short}.
 
+[id="ocp-4-17-installation-aws-outposts-deprecated_{context}"]
+==== Announcement of the deprecation of extending compute nodes into AWS Outposts for clusters deployed on AWS Public Cloud
+
+With this release, extending compute nodes into AWS Outposts for clusters deployed on AWS Public Cloud is deprecated. The ability to deploy compute nodes into AWS Outposts after installation, as an extension of an existing {product-title} cluster operating in a public AWS region, will be removed with the release of {product-title} version 4.20.
```

Comment:
Do we know for fact that this will happen? Future-casting is risky, just FYI.

Reply:
Normally I'd agree, but I got this wording directly from PM. thanks for calling it out